### ISSUE TITLE
Pi pico example: cannot find trait `DelayUs` in module `embedded_hal::delay`

### ISSUE THREAD

**Issue by gabevenberg** [rtic-rs/rtic#849]
When running `cargo check` or `cargo build` on the rp2040 example, (with rust nightly and the correct targets installed, etc), compilation of rtic-monotonics fails with the following compilation error:

```
error[E0405]: cannot find trait `DelayUs` in module `embedded_hal::delay`
   --> /home/gabe/personal/rtic/rtic-monotonics/src/rp2040.rs:165:27
    |
165 | impl embedded_hal::delay::DelayUs for Timer {
    |                           ^^^^^^^ help: a trait with a similar name exists: `DelayNs`
    |
   ::: /home/gabe/.cargo/registry/src/index.crates.io-6f17d22bba15001f/embedded-hal-1.0.0-rc.2/src/delay.rs:4:1
    |
4   | pub trait DelayNs {
    | ----------------- similarly named trait `DelayNs` defined here
    ```

**Comment by Finomnis** [comment]
Hey! Is this still the case? This should have been fixed in #843.

**Comment by gabevenberg** [comment]
seems to be fixed with new repo clone, yah.
